platforms/software/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/dsl/DefaultComponentMetadataHandlerTest.groovy

```diff
@@ -110,23 +110,6 @@ class DefaultComponentMetadataHandlerTest extends Specification {
         ruleWrapper.rule.spec == Specs.satisfyAll()
     }
 
-    def "add rule source rule that applies to all components"() {
```

Comment:
Is it worth leaving these tests in while the methods are still present?

If these were removed due to the deprecation logger not being initialized, we should initialize the deprecation logger in these tests instead of removing coverage